moved/src/move_execution/canonical.rs

```diff
@@ -82,12 +95,15 @@ pub(super) fn verify_transaction(
     Ok(())
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Let's create a `P2` issue to fix the too many args issue. We'll probably create a few structs for gas and execution context to wrap the inputs together.

Reply:
Created a new issue for it: https://github.com/MovedNetwork/op-move/issues/184